utility/doc/E2E_FEATURE_PATH_TEMPLATE.md

```diff
@@ -0,0 +1,218 @@
+
+**Deliverables**:
+
+**POC**:
+
+- [ ] A POC SPIKE to be closed out and split out into multiple PRs
+
+**MVP**:
+
+- [ ] A PR that adds or modifies relevant structures and interfaces; such as [shared/core/types/proto](../../shared/core/types/proto), [shared/modules](../../shared/modules), etc
+- [ ] A PR that materializes an MVP of the feature along with unit tests
+- [ ] A PR that introduces a new E2E tests with **one happy** and **one sad** path scenarios as described in the origin document (refer to [e2e/README.md](../../e2e/e2e/README.md)); this may require additions to the [cli](https://github.com/pokt-network/pocket/tree/main/app/client)
+- [ ] A PR that updates all pertinent documentation
+
+**PROD**:
+
+- [ ] One or more subsequent GitHub issues that track future work including, but not limited to:
+  - Enhancing test coverage
+  - Adding subsequent features
+  - Patching hacks or workarounds
+  - Enabling your [imagination](https://github.com/pokt-network/pocket/assets/1892194/6aff9004-8d3b-48e8-b6d5-9b67ac266e3d)!
+
+#### Origin Document
+
+**Purpose:** A single sentence that captures the intended purpose, behaviour and goal of the E2E feature
+**Actors**: Check all of the protocol actors involved in the feature:
+
+- [ ] Validator
+- [ ] Application
+- [ ] Servicer
+- [ ] Fisherman
+- [ ] Portal
+
+**Data Structures**:
+
+- A list of the core types (protobufs, structs, etc) that will be used, added or modified in this feature
+- Mention or link to specific files if applicable
+- See [shared/core/types/proto](../../shared/core/types/proto) as a reference as they will most likely, but not necessarily, be part of that package
+- _TIPS:_
+
+  - _This will be non-exhaustive and will likely change during the POC or MVP stages_
+  - You can find all other structs by running this command:
+
+    ```bash
+    grep -r "type .* struct" --exclude-dir="vendor" --exclude="*.gen.go" --exclude="*.pb.go" .
+    ```
+
+  - You can find all other protobufs by running this command:
+
+    ```bash
+    find . -name "*.proto" -not -path "./vendor/*"`
+    ```
+
+**Interfaces**:
+
+- A list of the interface (go interfaces, placeholder functions, grpc, etc) that will be used, added or modified in this feature
+- Mention or link to specific files if applicable
+- See [shared/modules](../../shared/modules) as a reference as they will most likely, but not necessarily, be part of that package
+- _TIPS:_
+
+  - _This will be non-exhaustive and will likely change during the POC or MVP stages_
+  - You can find all other structs by running this command:
+
+    ```bash
+    grep -r "type .* interface" --exclude-dir="vendor" --exclude="*.gen.go" --exclude="*.pb.go" .
+    ```
+
+  - You can find all other protobufs by running this command:
+
+    ```bash
+    find . -name "*.proto" -not -path "./vendor/*"`
+    ```
+
+**Diagram**:
+
+- _One or more mermaid diagrams that will visualize the E2E feature_
```

Comment:
Is there a particular reason for this line to be italicized?

Reply:
Nope :)